local/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	hookdeck "github.com/hookdeck/hookdeck-go-sdk"
+	hookdeckclient "github.com/hookdeck/hookdeck-go-sdk/client"
```

Comment:
got these errors locally

![CleanShot 2023-09-02 at 00 02 50](https://github.com/hookdeck/hookdeck-api-schema/assets/26750127/741da61b-ce35-43ba-ac84-32100da6e1b8)

Reply:
I assume because of this, VSCode couldn't infer the types so I don't have any auto complete while writing code in this file